config/env/all.js

```diff
@@ -5,5 +5,5 @@ var keys = rootPath + '/keys.txt';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-  db: process.env.DB_URL
+    db: process.env.MONGOHQ_URL
```

Comment:
@andela-f-opara Please you need to make use of DB_URL, so we are all in sync in the name we use for our variables. The variable names need to be in the sam every where but the value could be anything